renderer/components/js/components.js

```diff
@@ -60,19 +70,21 @@ org.ekstep.questionunit.questionComponent = {
                 </div>\
             </div>\
             <div class="expand-button" onclick="org.ekstep.questionunit.questionComponent.toggleQuestionText()">\
-                <img src="" id="org-ekstep-contentrenderer-questionunit-questionComponent-downArwImg"/>\
+                <img class="exp-button" src="" id="org-ekstep-contentrenderer-questionunit-questionComponent-downArwImg"/>\
```

Comment:
Write onload function for this & call org.ekstep.questionunit.questionComponent.onDomReady();

Remove the below script attached to this template.
<script>org.ekstep.questionunit.questionComponent.onDomReady();</script>